pkg/distributor/forwarding/forwarding_test.go

```diff
@@ -109,7 +109,7 @@ func TestForwardingSamplesSuccessfullyToSingleTarget(t *testing.T) {
 	cortex_distributor_forward_requests_total{} 1
 	# HELP cortex_distributor_forward_samples_total The total number of samples the Distributor forwarded.
 	# TYPE cortex_distributor_forward_samples_total counter
-	cortex_distributor_forward_samples_total{} 4
+	cortex_distributor_forward_samples_total 4
```

Comment:
Change like this is unnecessary and this file still has multiple `{}` in expected metrics strings. (I'd suggest sending cleanups like this in separate PR)

Reply:
ack, removed both